src/backend/src/routes/projects.routes.ts

```diff
@@ -11,10 +11,29 @@ projectRouter.get('/link-types', ProjectsController.getAllLinkTypes);
 projectRouter.get('/:wbsNum', ProjectsController.getSingleProject);
 projectRouter.post(
   '/create',
-  intMinZero(body('crId')),
   nonEmptyString(body('name')),
+  intMinZero(body('crId')),
   intMinZero(body('carNumber')),
+  body('teamIds').isArray(),
```

Comment:
these vaidations should be a function that weuse in both create and edit, maybe pass in an isOptional boolean to add the optional things